packages/gatsby-plugin-cxs/.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "presets": [
+    ["../../.babel-preset.js", {
```

Comment:
```suggestion
    ["babel-preset-gatsby-package", {
```

we recently added this, so sorry for the churn here!

Reply:
No worry 😄